client/core/core.go

```diff
@@ -36,6 +36,9 @@ const (
 	keyParamsKey      = "keyParams"
 	conversionFactor  = 1e8
 	regFeeAssetSymbol = "dcr" // Hard-coded to Decred for registration fees, for now.
+	// high number to be assigned to 'regConfirms' in 'dexConnection'
+	// when the registration is completed
+	regConfirmationsPaid uint32 = 9999
```

Comment:
The 'PNOOMA' number used to represent that the confirmation is completed.

Reply:
OK, although note this is actually realistic for many ccs.  e.g. ETH: https://blog.coinbase.com/announcing-new-confirmation-requirements-4a5504ba8d81